lib/spaceship/tunes/language_converter.rb

```diff
@@ -28,23 +28,24 @@ def from_itc_to_itc_readable(from)
         end
 
         private
+
           # Path to the gem to fetch resoures
-          def spaceship_gem_path
-            if Gem::Specification::find_all_by_name('spaceship').any?
-              return Gem::Specification.find_by_name('spaceship').gem_dir
-            else
-              return './'
-            end
+        def spaceship_gem_path
```

Comment:
I too, prefer aligning `private` methods as well, but apparently most ruby projects prefer not aligning them.